### PR TITLE
update 1.2.0

### PR DESCRIPTION
**Pull request by Arishamays1**
## Joblib 1.2.0 Update
[Jira Ticket](https://anaconda.atlassian.net/browse/PKG-2044)
[Upstream](https://github.com/joblib/joblib)
[Dependencies ](https://github.com/joblib/joblib/blob/master/pyproject.toml)
# Changes
- Updated version number and `sha256`
- Updated pinnings on dependencies

### CODE REVIEW

**Review by boldorider4 — APPROVED** [review]

File: recipe/meta.yaml [hunk 1]
```diff
@@ -49,10 +49,8 @@ about:
     large data in particular and has specific optimizations for numpy arrays."
   license: BSD-3-Clause
   license_family: BSD
-  license_file: LICENSE.txt
   license_url: https://github.com/joblib/joblib/blob/{{ version }}/LICENSE.txt
   doc_url: https://joblib.readthedocs.io/en/latest/
```
Comment (Jrice1317) [inline recipe/meta.yaml:53]:
```suggestion
  doc_url: https://joblib.readthedocs.io/
```

Comment (Jrice1317) [reply recipe/meta.yaml:53]: Maybe it's best to change these urls for those whose native language is not English.

Comment (Jrice1317) [reply recipe/meta.yaml:53]: Maybe use the same url for the `home` url as well.

`home: https://joblib.readthedocs.io/`

**Review by Jrice1317 — COMMENTED** [review]
LGTM

File: recipe/meta.yaml (hunk 1 above)
Comment (Arishamays1) [reply recipe/meta.yaml]: Makes sense! Updated. Thanks!

File: recipe/meta.yaml [hunk 2]
```diff
@@ -49,10 +49,8 @@ about:
     large data in particular and has specific optimizations for numpy arrays."
   license: BSD-3-Clause
   license_family: BSD
-  license_file: LICENSE.txt
   license_url: https://github.com/joblib/joblib/blob/{{ version }}/LICENSE.txt
```
Comment (skupr-anaconda) [inline recipe/meta.yaml:52]: Could you remove license_url instead of license_file?

**Pull request merged**